ui/main/src/app/business/services/card/selectedCard.service.ts

```diff
@@ -60,6 +62,15 @@ export class SelectedCardService {
     public getSelectCard(): Observable<SelectedCard> {
         return this.selectedCardWithChildrenChange.asObservable();
     }
+
+    public setSelectedCardDeleted(cardId: string) {
```

Comment:
i would name it setCardDeleted as it is not always the selected card that is deleted